tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field_semantic_analysis_negative.bal

```diff
@@ -71,6 +71,66 @@ function testFieldWithNeverType() {
     Foo _ = {...rec5, ...rec6};
 }
 
+type Address record {
+    string street;
+};
+
+type Street record {|
+    string street;
+|};
+
+type Foz record {|
+    string s;
+    int i;
+    int j;
+    boolean b;
+    string...;
+|};
+
+function testSpreadOfOpenRecordToCreateClosedRecord() {
+    Address address1 = {street: "Main Street"};
+    record {|string street;|} _ = {...address1};
+
+    Address address2 = {street: "Pine Street", "number": 40};
```

Comment:
Since these are compile-time errors, this test doesn't really test anything extra compared to the first test, right?